packages/plugins/src/plugins/glint-report.plugin.ts

```diff
@@ -37,10 +41,32 @@ export class GlintReportPlugin implements Plugin<PluginOptions> {
     return Promise.resolve([fileName]);
   }
 
+  getDiagnostics(service: Service, fileName: string): DiagnosticWithContext[] {
+    const languageService = service.getLanguageService();
+    const program = languageService.getProgram()!;
+    const checker = program.getTypeChecker();
+
+    const diagnostics = service.getDiagnostics(fileName);
```

Comment:
I think you want to go through this https://github.com/rehearsal-js/rehearsal-js/blob/master/packages/codefixes/src/get-diagnostics.ts#L26

Reply:
Good call, fixed.